app/assets/js/rtMedia.backbone.js

```diff
@@ -1005,8 +1005,14 @@ jQuery( document ).ready( function( $ ) {
 
 			var rt_uploader_div = whats_new_form.find( '.rtmedia-uploader-div' );
 			var rt_uploader_filelist = whats_new_form.find( '#rtmedia_uploader_filelist' );
+			var whats_new_option = whats_new_form.find( '#whats-new-options' );
 
 			rt_uploader_div.show();
+			whats_new_option.show();
```

Comment:
@dishitpala Before performing actions, should we check length of the object to make sure it exits?

Reply:
make sense 👍